### ISSUE TITLE
Compatibility with Excel 2003 & earlier (.ppt)

### ISSUE THREAD

**Issue by Bugro** [scanny/python-pptx#362]

**Comment by MartinPacker** [comment]
I seriously doubt you'll get this as python-pptx is manipulating the XML - which .ppt doesn't have. I wonder if you can automate the conversion between ppt and pptx some other way.

And why would you want ppt?

**Comment by DZemens** [comment]
@MartinPacker right! For one-off applications, just open & re-save manually in the new format. The conversion occurs automatically when opening the .ppt format, IIRC. 

If there's a large volume of files that need conversion, I haven't tried this yet but seems like you could do it in .bat, or in python with win32com, or if all else fails, it would be trivial in plain old VBA.
